Regression/SGD/src/main/java/org/tribuo/regression/sgd/TrainTest.java

```diff
@@ -86,22 +86,22 @@ public String getOptionsDescription() {
         /**
          * Number of SGD epochs. Defaults to 5.
          */
-        @Option(charName = 'i', longName = "epochs", usage = "Number of SGD epochs. Defaults to 5.")
+        @Option(charName = 'i', longName = "epochs", usage = "Number of SGD epochs.")
```

Comment:
Are these defaults now displayed in a user-visible way elsewhere? I know that we don't consistently record them in usage strings, but I don't think that means that we should remove them from places where they do appear - they're important for users who aren't looking at the source code.

Reply:
Yes, the options system prints them when you ask for usage:
```
CARTClassificationOptions

Description: Options for decision/classification trees.
Char Long Name                  Type                                             Default Usage
     cart-impurity              enum - {GINI, ENTROPY}                           GINI    Impurity measure to use. Defaults to GINI.
     cart-max-depth             int                                              6       Maximum depth in the decision tree.
     cart-min-child-weight      float                                            5.0     Minimum child weight.
     cart-min-impurity-decrease float                                            0.0     Minimum impurity decrease.
     cart-print-tree            boolean                                          false   Prints the decision tree.
     cart-random-split          boolean                                          false   Whether to choose split points for features at random.
     cart-seed                  long                                             12345   RNG seed.
     cart-split-fraction        float                                            1.0     Fraction of features in split.
     cart-tree-algorithm        enum - {CART}                                    CART    Tree algorithm to use (options are CART).
```